<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Rest Api Test Automation Test Suite" duration-ms="1855" started-at="2022-08-24T09:24:01Z" finished-at="2022-08-24T09:24:03Z">
    <groups>
    </groups>
    <test name="Rest Api Test Cases" duration-ms="1855" started-at="2022-08-24T09:24:01Z" finished-at="2022-08-24T09:24:03Z">
      <class name="restApi.qa.test.GetApiTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:restApi.qa.test.GetApiTest@36d4b5c]" name="setUp" is-config="true" duration-ms="10" started-at="2022-08-24T05:24:01Z" finished-at="2022-08-24T05:24:01Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="getApiTest()[pri:1, instance:restApi.qa.test.GetApiTest@36d4b5c]" name="getApiTest" duration-ms="1410" started-at="2022-08-24T05:24:01Z" finished-at="2022-08-24T05:24:03Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Status code is 429 expected [200] but found [429]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Status code is 429 expected [200] but found [429]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at restApi.qa.test.GetApiTest.getApiTest(GetApiTest.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getApiTest -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:restApi.qa.test.GetApiTest@36d4b5c]" name="setUp" is-config="true" duration-ms="0" started-at="2022-08-24T05:24:03Z" finished-at="2022-08-24T05:24:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="getEmployeePersonalDataTest()[pri:2, instance:restApi.qa.test.GetApiTest@36d4b5c]" name="getEmployeePersonalDataTest" duration-ms="399" started-at="2022-08-24T05:24:03Z" finished-at="2022-08-24T05:24:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getEmployeePersonalDataTest -->
      </class> <!-- restApi.qa.test.GetApiTest -->
    </test> <!-- Rest Api Test Cases -->
  </suite> <!-- Rest Api Test Automation Test Suite -->
</testng-results>
